server/src/main/java/com/google/codeu/servlets/StatsPageServlet.java

```diff
@@ -1,20 +1,24 @@
 package com.google.codeu.servlets;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.JsonObject;
 import java.io.IOException;
-
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/**
- * Responds with a hard-coded message for testing purposes.
- */
+/* Responds with a hard-coded message for testing purposes. */
```

Comment:
Nit add an extra * to your beginning comment.

Should look like `/**  Responds with a hard-coded message for testing purposes. */`